src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/service/GroupsService.kt

```diff
@@ -31,8 +32,14 @@ class GroupsService(
   private val maintainUserCheck: MaintainUserCheck,
 ) {
 
+  companion object {
+    private val CRS_GROUP_CODE_PREFIXES = listOf("INT_CR_", "INT_SP_")
+  }
+
   suspend fun getAllGroups() = groupRepository.findAllByOrderByGroupName()
 
+  suspend fun getAllCRSGroups() = groupRepository.findAllByOrderByGroupName().filter { it.groupCode.startsWithOneOf(*CRS_GROUP_CODE_PREFIXES.toTypedArray()) }
```

Comment:
This could be simplified and wouldn't need the custom method:
```suggestion
  suspend fun getAllCRSGroups() = getAllGroups().filter { CRS_GROUP_CODE_PREFIXES.any(it.groupCode::startsWith) }
```